### PR TITLE
Kalfon

### PR DESCRIPTION
**Pull request by jkobject**

### CODE REVIEW

**Comment by jkobject** [comment]
renaming some import for movies.py
changing comments, removing a test and the ''' into """ for rois
preprocessing, only changing the location of the functions

**Comment by jkobject** [comment]
oups something went wrong with the nosetests. and I forgot to add assertions.

**Comment by jkobject** [comment]
this is an error that I am not able to understand.

**Comment by jkobject** [comment]
I have removed only commented parts.(approved by Andrea)

File: caiman/source_extraction/cnmf/initialization.py [hunk 1]
```diff
@@ -451,51 +469,54 @@ def greedyROI(Y, nr=30, gSig=[5, 5], gSiz=[11, 11], nIter=5, kernel=None, nb=1):
...
+    @Author: Eftychios A. Pnevmatikakis based on a matlab implementation by Yuanjun Gao
             Simons Foundation, 2015
+
+    See Also:
+    -------
+    http://www.cell.com/neuron/pdf/S0896-6273(15)01084-3.pdf
+
+
     """
-    debug_ = False
```
Comment (agiovann) [inline caiman/source_extraction/cnmf/initialization.py]: why did you erase this portion?

Comment (agiovann) [inline caiman/source_extraction/cnmf/initialization.py]: OK, it is not super important

File: caiman/utils/utils.py [hunk 2]
```diff
@@ -22,22 +32,45 @@
...
-        if not(os.path.exists('./example_movies/demoSue2x.tif')):        
-            url = 'https://www.dropbox.com/s/09z974vkeg3t5gn/Sue_2x_3000_40_-46.tif?dl=1'
-            print("downloading demo Sue2x with urllib")
-            f = urlopen(url)
-            data = f.read()
-            with open("./example_movies/demoSue2x.tif", "wb") as code:
-                code.write(data)
-        else:
-            print('File already existing')
+        for f in file_list:
```
Comment (agiovann) [inline caiman/utils/utils.py]: here I would not force to download all the files if not required...

**Comment by jkobject** [comment]
Seen your comment, 
I will prepare a reply for tomorrow. 
Most of it is just entirely due to old commits that I have not managed well 
when you ignore files with gitignore but you have already pushed them before.
Will repair all of that.
Have a good night.

**Comment by jkobject** [comment]
everything should be fine, I have manually removed the files that haven't due to the git ignore.
I had already taken in account johannes change, I don't see any modification in the install file except for two comments that help the user and the shell understand what it is.

**Comment by jkobject** [comment]
It should work better now.

**Comment by j-friedrich** [comment]
Hi Jeremie. I made some more cumbersome changes to visualization.py to use the fairly recent Bokeh array protocol. That enabled to fix the problem with the max-projected plot for 3D data and made indeed all Bokeh plots much faster to plot and interact. Committed directly to master, you might have to check whether your code is compatible and merge my changes to yours.

**Pull request merged**